spec/controllers/oauth/applications_controller_spec.rb

```diff
@@ -160,13 +162,15 @@ module Oauth
     it "should not let a user get the list of his applications" do
```

Comment:
Since the HTTP status is now 200 should the "not" be in this spec title?

Reply:
The user does not get a list of applications, just the heading which is why I didn't change the name.

Reply:
On further review, I no longer think this is a valid test. The user in the test is part of the member_ids so it will not fail as it previously did.